python/activator/middleware_interface.py

```diff
@@ -176,6 +181,24 @@ def __init__(self, central_butler: Butler, image_bucket: str, instrument: str,
         # How much to pad the refcat region we will copy over.
         self.padding = 30*lsst.geom.arcseconds
 
+    def _get_deployment(self):
+        """Get a unique version ID of the active stack and pipeline configuration(s).
+
+        Returns
+        -------
+        version : `str`
+            A unique version identifier for the stack. Contains only
+            word characters and "-", but not guaranteed to be human-readable.
+        """
+        try:
+            return os.environ["K_REVISION"]
```

Comment:
I'm trying to keep the docstring agnostic to what is actually used as the basis of the deployment label, especially since I think we might want to change it in the future. Would a code comment suffice?

Reply:
Yes I think so